examples/nuxt/nuxt-headlesswp-gravity-forms/wp-env/.wp-env.json

```diff
@@ -0,0 +1,23 @@
+{
+  "phpVersion": "7.4",
+  "plugins": [
+    "https://github.com/wp-graphql/wp-graphql/releases/latest/download/wp-graphql.zip",
+    "https://github.com/AxeWP/wp-graphql-gravity-forms/releases/download/v0.13.0.1/wp-graphql-gravity-forms.zip"
```

Comment:
@Fran-A-Dev 

Can we change this to `https://github.com/AxeWP/wp-graphql-gravity-forms/releases/latest/download/wp-graphql-gravity-forms.zip` as this will always get the latest version or would prefer to keep this to always version 13?